tensorflow/c/experimental/ops/math_ops.cc

```diff
@@ -45,7 +45,13 @@ Status Conj(AbstractContext* ctx,
       DataTypeIsInteger(BaseType(dtype))) {
     TF_RETURN_IF_ERROR(Identity(ctx, inputs, outputs, name));
   } else {
```

Comment:
Could we do this only for complex and variant types as in [python](https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/python/ops/math_ops.py;l=3982-3988;drc=8910f7db3653e09386fb9cf0d0d8f7c9fcf0bc8c)?

Something like 

```
auto dtype = BaseType(inputs[0]->DataType());

if (DataTypeIsFloating(dtype) || DataTypeIsInteger(dtype)) {
  ...
} else if (DataTypeIsComplex(v) || v == DT_VARIANT) {
  // Your code
} else {
  return errors::InvalidArgument("Expected numeric or variant tensor, got dtype ",  dtype);
}
```

Reply:
Done